dspace-api/src/main/java/org/dspace/administer/CreateAdministrator.java

```diff
@@ -103,20 +126,20 @@ protected CreateAdministrator()
      *
      * @throws Exception if error
      */
-    protected void negotiateAdministratorDetails()
-        throws Exception {
+    protected void negotiateAdministratorDetails(CommandLine line)
+            throws Exception {
         Console console = System.console();
 
         System.out.println("Creating an initial administrator account");
 
-        boolean dataOK = false;
-
         String email = null;
         String firstName = null;
         String lastName = null;
-        char[] password1 = null;
-        char[] password2 = null;
         String language = I18nUtil.getDefaultLocale().getLanguage();
+        ConfigurationService cfg = DSpaceServicesFactory.getInstance().getConfigurationService();
+        boolean flag = line.hasOption('p');
+        char[] password = null;
+        boolean dataOK = !(!line.hasOption('f') || !line.hasOption('e') || !line.hasOption('l'));
```

Comment:
By deMorgan's Rule this reduces to `boolean dataOK = line.hasOption('f') && line.hasOption('e') && line.hasOption('l');` which is a bit easier to understand.

Reply:
@mwoodiupui . I resolved this